libraries/DataFlash/DataFlash_MAVLinkLogTransfer.cpp

```diff
@@ -298,6 +298,7 @@ bool DataFlash_Class::handle_log_send_data()
     _log_data_remaining -= len;
     if (ret < 90 || _log_data_remaining == 0) {
         _log_sending = false;
+        _in_log_download = false;
```

Comment:
After all data has been send we reset _in_log_download variable, so new log could be started

Reply:
When this error occurs, how does it show itself to the user?  I'd like to cherry-pick this fix in...